<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ans</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_val">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>key</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>val</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_s_add">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>add to set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_s_check">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>is in set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_m_add">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>add to map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_m_check">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>is in map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_m_get">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>map: get val</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_s_clear">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>clear set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_s_remove">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>remove from set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_m_remove">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>remove from map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_m_clear">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>clear map</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_set">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_map">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>331</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Map</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
